aggregator.ts

```diff
@@ -189,7 +189,6 @@ namespace jacdac {
             this.handleRegInt(packet, SensorAggregatorReg.NumSamples, this.numSamples)
             this.handleRegInt(packet, SensorAggregatorReg.SampleSize, this.sampleSize)
             this.streamSamples = this.handleRegInt(packet, SensorAggregatorReg.StreamSamples, this.streamSamples)
```

Comment:
this is not the system reg, by design? @mmoskal

Reply:
It could be system reg. The only thing it should be more than 8 bits.